ssofact.routing.yml

```diff
@@ -5,3 +5,12 @@ ssofact.redirect_login:
     _title: 'OpenID Connect ssoFACT login form redirect page'
   requirements:
     _custom_access: '\Drupal\ssofact\Controller\SsofactRedirectController::access'
+ssofact.user:
+  path: '/user/sso'
+  defaults:
+    _controller: '\Drupal\ssofact\Controller\Ssofact::content'
+    _title: 'ssoFACT'
+  requirements:
+    _permission: 'change own username'
```

Comment:
This also isn't clear to me. Why is it a requirement to access the URL?

Reply:
Because otherwise also anonymous users would have access to the route.

Reply:
Why not use `role` as `authenticated` for this? IMHO will be more clear... https://www.drupal.org/docs/8/api/routing-system/structure-of-routes

Reply:
This page should only display placeholder strings for the page title and page content.

Where is the requirement coming from that this path should only be accessible for authenticated users?

The output of this page is simply used as an HTML page template wrapper for all output generated by the SSO web application.  The main pages generated by the SSO are the login page, registration page, and password reset page (as also listed in the Asana task description).  All three of them are dealing with anonymous users.

Reply:
Yeah, did not think about this… I thought it should be for authenticated users only as it's under the `/user` route. But thinking of this it doesn't make much sense, sorry.